### ISSUE TITLE
Azure: error /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `CXXABI_1.3.9'

### ISSUE THREAD

**Issue by Soudny** [Microsoft/napajs#219]
Hey,
First of all, I would like to say that I have tried the following solution for this issue without any luck:
https://github.com/Microsoft/napajs/wiki/FAQ#linux

So basically even though this might solve the issue, Azure keeps telling me that libstdc++6 is at its latest version and simply refuses to do anything. Keep in mind that in Azure Web App, I do not have sudo command whatsoever.

Is there any workaround to making this work? Why wouldn't this start if libstdc++6 is already there?

Anyone else had this issue?

**Comment by fs-eire** [comment]
I would like to take a look on this issue. Could you share me how can I reproduce it?

**Comment by Soudny** [comment]
Of course, it's quite simple actually - just create a Web App on Azure with linux selected as OS. I used instances hosted on US East / US East 2, but it shouldn't matter too much.

Yesterday, I heard a response from Microsoft support stating that node containers there use Debian 8.10 (Jessie) with gcc version 4.9.2 by default.

That, unfortunately, does not seem to be enough and will require using Docker with Debian 9 (Stretch) with node 8.11 in it (node 9+ throws the same error even though all the necessary compilers are there).

To be clear, I still haven't tried this fix, however they assure me that it will work.
